tests/system/providers/google/cloud/dataflow/example_dataflow_sql.py

```diff
@@ -26,6 +26,8 @@
 from airflow import models
 from airflow.providers.google.cloud.operators.dataflow import DataflowStartSqlJobOperator
 
+DAG_ID = "example_gcp_dataflow_sql"
+
 GCP_PROJECT_ID = os.environ.get("GCP_PROJECT_ID", "example-project")
 
 BQ_SQL_DATASET = os.environ.get("GCP_DATAFLOW_BQ_SQL_DATASET", "airflow_dataflow_samples")
```

Comment:
Thank you for the work done!
I was checking this example dag in my env and it failed because of lack of such table in my project. 
For now, the google team is working on making all the system tests self contained as possible, to ensure that the execution of the system tests will be successful for all users and when running in the CI. That is why we are trying to create/delete all the resources that will be used directly in the system test. In this case it will be better to create such BigQuery table right inside the Dag, upload some dummy info and delete this table after execution of Dataflow SQL query. It will ensure that the test will execute successfully no matter what :)